drivers/storage/isilon/executor/isilon_executor.go

```diff
@@ -43,6 +44,18 @@ func (d *driver) Name() string {
 	return isilon.Name
 }
 
+// Supported returns a flag indicating whether or not the platform
+// implementing the executor is valid for the host on which the executor
+// resides.
+func Supported(ctx types.Context, opts types.Store) (bool, error) {
+	// make sure mount.nfs is present
+	if err := exec.Command(
```

Comment:
Your logic works for me :) You could also use [this](https://github.com/akutz/gotil/blob/master/gotil.go#L165).